backend/src/main/java/ca/gov/dtsstn/cdcp/api/web/json/JsonMergePatchHttpMessageConverter.java

```diff
@@ -38,8 +40,8 @@ protected JsonMergePatch readInternal(Class<? extends JsonMergePatch> clazz, Htt
 
 	@Override
 	protected void writeInternal(JsonMergePatch jsonMergePatch, HttpOutputMessage httpOutputMessage) throws IOException, HttpMessageNotWritableException {
-		try {
-			Json.createWriter(httpOutputMessage.getBody()).write(jsonMergePatch.toJsonValue());
+		try (JsonWriter jsonWriter = Json.createWriter(httpOutputMessage.getBody())){
```

Comment:
```suggestion
		try (final var jsonWriter = Json.createWriter(httpOutputMessage.getBody())){
```

Reply:
Implemented in commit 74e3d8d4dbedcbce2497740fdcf756f4ac2d3d92.